inc/computer.class.php

```diff
@@ -212,5 +214,118 @@ static function showMassiveActionsSubForm(MassiveAction $ma) {
       return parent::showMassiveActionsSubForm($ma);
    }
 
+   /**
+    * Hack of PluginFieldsContainer::getAddSearchOptions
+    * I remove part of SQL restrict on profile
+    * when cron run, the offical function return nothing because not have profile
+    */
+   function pluginFieldsGetAddSearchOptions($itemtype, $containers_id = false) {
+      global $DB;
+
+      $opt = [];
+
+      $i = 76665;
+
+      $query = "SELECT DISTINCT fields.id, fields.name, fields.label, fields.type, fields.is_readonly,
```

Comment:
Using dbiterator would be much better.

Reply:
Yes, the goal is not rewrite all the plugin field function, but copy it here and remove the 3 lines.
I will propose a complete patch in the plugin fields repository soon